docs/yul.rst

```diff
@@ -288,6 +288,7 @@ variables at the same time. For this, the number and types of the
 values have to match.
 If you want to assign the values returned from a function that has
 multiple return parameters, you have to provide multiple variables.
+Assigning to the same variable multiple times, e.g. ``x, x := f()`` is invalid.
```

Comment:
As it stands, it is ambiguous in my opinion and the example is the only way to correctly understand what is meant.
Maybe either say "is the same statement" or use the same formulation as in the changelog.